<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGpxProper</class>
 <widget class="QDialog" name="CGpxProper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Track-Eigenschaften</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="l_GpxPath">
     <property name="text">
      <string notr="true">TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Name</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="le_Name"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Start &amp; R체ckw채rts</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QDateTimeEdit" name="dte_Start">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_Rev">
            <property name="text">
             <string>R체ckw채rts</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Farbe</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="rb_StdCol">
            <property name="text">
             <string>Standard</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_FlugCol">
            <property name="text">
             <string>Flug</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_col">
            <property name="text">
             <string>:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_Col">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pb_Bck">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string notr="true">&lt;&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_Nxt">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string notr="true">&gt;&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CGpxProper</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CGpxProper</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Col</sender>
   <signal>clicked()</signal>
   <receiver>CGpxProper</receiver>
   <slot>OnCol()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Bck</sender>
   <signal>clicked()</signal>
   <receiver>CGpxProper</receiver>
   <slot>OnBck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Nxt</sender>
   <signal>clicked()</signal>
   <receiver>CGpxProper</receiver>
   <slot>OnNxt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_col</sender>
   <signal>toggled(bool)</signal>
   <receiver>CGpxProper</receiver>
   <slot>OnDefColChg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnCol()</slot>
  <slot>OnBck()</slot>
  <slot>OnNxt()</slot>
  <slot>OnDefColChg()</slot>
 </slots>
</ui>
